client/site/site.less

```diff
@@ -663,6 +663,33 @@ body.welcome {
       }
     }
   }
+
+  .donation-banner {
+    display: flex;
+    margin-top: -50px;
+    height: 50px;
+
+    a {
```

Comment:
This link looks better with font-weight: bold and color: #333 (or something like that).